cmd/scollector/collectors/stream.go

```diff
@@ -0,0 +1,82 @@
+package collectors
+
+import (
+	"reflect"
+	"runtime"
+	"sync"
+	"time"
+
+	"bosun.org/collect"
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+)
+
+type StreamCollector struct {
```

Comment:
I'm not sure the stream collector is really needed here. Why not just have the listener relay data points to the collect package directly. That channel is made here: https://github.com/bosun-monitor/bosun/blob/master/cmd/scollector/main.go#L191. I would suggest putting the http listener in the main package and relaying directly to collect.